DataModel/Protocol/Protocol-definition.yml

```diff
@@ -0,0 +1,285 @@
+    microscope_setup_description:
+      description: Description of the microscope setup
+      type: string
+      format: text
+      ontology_link: https://bioportal.bioontology.org/ontologies/OBI?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FOBI_0400169
+    microscopy_technique:
+      description: The type of microscopy
+      type: string
+      format: text
+      ontology_link: https://bioportal.bioontology.org/ontologies/SNOMEDCT?p=classes&conceptid=http%3A%2F%2Fpurl.bioontology.org%2Fontology%2FSNOMEDCT%2F278289002
+    magnification:
+      description: Magnification of the objective used for imaging
+      type: string
+      format: text
+    numerical_aperture:
+      description: Numerical aperture of the objective
+      type: integer
+      format: number
+      ontology_link: https://bioportal.bioontology.org/ontologies/PLOSTHES?p=classes&conceptid=http%3A%2F%2Flocalhost%2Fplosthes.2017-1%2310222
+    immersion_medium_type:
+      description: Immersion medium used for imaging
+      type: string
+      format: text
+      ontology_link: https://bioportal.bioontology.org/ontologies/NCIT?p=classes&conceptid=http%3A%2F%2Fncicb.nci.nih.gov%2Fxml%2Fowl%2FEVS%2FThesaurus.owl%23C182013
+    immersion_medium_refractive_index:
+      description: Refractive index of the immersion medium used for imaging
+      type: integer
+      format: number
+      ontology_link: https://bioportal.bioontology.org/ontologies/FBbi?p=classes&conceptid=http%3A%2F%2Fpurl.obolibrary.org%2Fobo%2FFBbi_00000311
+    pixel_size:
+      description: Pixel size in nanometers
+      type: integer
+      format: nm
+      ontology_link: https://bioportal.bioontology.org/ontologies/NCIT?p=classes&conceptid=http%3A%2F%2Fncicb.nci.nih.gov%2Fxml%2Fowl%2FEVS%2FThesaurus.owl%23C94953
+    number_of_tiles:
+      description: Number of XY tiles in the experiment
+      type: integer
+      format: number
+    tile_size_y:
+      description: ' Y size of the tile in micrometers'
+      type: integer
+      format: um
+    tile_size_x:
+      description: X size of the tile in micrometers
+      type: integer
+      format: um
+    z_stack_step_size:
+      description: Z-stack step size in nanometers
+      type: integer
+      format: nm
+      ontology_link: https://bioportal.bioontology.org/ontologies/DCM?p=classes&conceptid=http%3A%2F%2Fdicom.nema.org%2Fresources%2Fontology%2FDCM%2F110903
+    overlapping_tiles:
+      description: Whether tiles were collected with overlap
+      type: boolean
+      format: boolean
+      options:
+      - 'True'
+      - 'False'
+    channel:
+      description: Information about each channel used in the imaging protocol
+      type: array
+      format: text
+      ontology_link: https://bioportal.bioontology.org/ontologies/BAO?p=classes&conceptid=http%3A%2F%2Fwww.bioassayontology.org%2Fbao%23BAO_0000459
```

Comment:
This ontology link redirects to the label `scanning probe microscopy`. Not sure how is this relevant to `channel` attribute